### PR TITLE
Separate snippet decoration

### PR DESCRIPTION
**Pull request by mlsteele**

### CODE REVIEW

**Comment by mlsteele** [comment]
@joshblum can you look at the go side? The goal is to move the 💣 slightly to the right.

File: go/chat/utils/utils.go [hunk 1]
```diff
@@ -819,34 +819,34 @@ func getSenderPrefix(mvalid chat1.MessageUnboxedValid, conv chat1.ConversationLo
...
 	var explodingPrefix string
 	if mvalid.IsEphemeral() {
-		explodingPrefix = "💣 "
+		decoration = "💣"
 	}
 
 	switch msg.GetMessageType() {
 	case chat1.MessageType_TEXT:
-		return explodingPrefix + senderPrefix + msg.Valid().MessageBody.Text().Body
+		return explodingPrefix + senderPrefix + msg.Valid().MessageBody.Text().Body, decoration
```
Comment (joshblum) [inline go/chat/utils/utils.go]: we can remove `explodingPrefix` throughout

File: go/chat/utils/utils.go [hunk 2]
```diff
@@ -860,7 +859,8 @@ func GetDesktopNotificationSnippet(conv *chat1.ConversationLocal, currentUsernam
 	}
 	mvalid := msg.Valid()
 	if !mvalid.IsEphemeral() {
-		return GetMsgSnippet(msg, *conv, currentUsername)
+		snippet, _ := GetMsgSnippet(msg, *conv, currentUsername)
```
Comment (joshblum) [inline go/chat/utils/utils.go]: we should use the decoration at L872 so it's consistent throughout -- why doesn't this snippet need the separation in general?

Comment (mlsteele) [reply go/chat/utils/utils.go]: what alignment changes?

**Comment by buoyad** [comment]
@keybase/react-hackers I added some styling to the snippet decoration
desktop: 
image
mobile: 
image

**Pull request merged**